controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -761,6 +784,11 @@ func saveHostProvisioningSettings(host *metal3v1alpha1.BareMetalHost) (dirty boo
 		dirty = true
 	}
 
+	if !reflect.DeepEqual(host.Status.Provisioning.RAID, host.Spec.RAID) {
+		host.Status.Provisioning.RAID = host.Spec.RAID.DeepCopy()
```

Comment:
We don't want to just copy what we're given here. We want to copy the parts of it we're actually going to use. For example, if both hardware and software RAID are configured we ignore the software settings, right? So we should not copy those. Are there other examples of settings we ignore?

Reply:
Thanks, done.